.github/workflows/php.yml

```diff
@@ -0,0 +1,30 @@
+name: PHP Composer
```

Comment:
Changing such CI setup is totally unrelated to passing timeouts to RemoteWebDriver. There is no reason to include that in the same PR IMO.

Reply:
Of course. Just played around with Github action and didn't want to include that in this PR. Will remove that. Sorry for that.